examples/advdiff-tpe.py

```diff
@@ -77,6 +77,13 @@ def main(actx_class, use_overintegration=False, use_esdg=False,
     if casename is None:
         casename = "mirgecom"
 
+    try:
+        from grudge.discretization import PartID  # noqa: F401
+    except ImportError:
+        from warnings import warn
+        warn("This example requires a coupling-enabled branch of grudge; exiting.")
+        return
```

Comment:
Curious...  why would this example require multiple volumes?

Reply:
should be fixed in [0238cbe](https://github.com/illinois-ceesd/mirgecom/pull/1048/commits/0238cbe73746c2fc3109a5456f581c621e8e302f)